### PR TITLE
Fix typo for Copy-GPO.MD

### PR DESCRIPTION
**Pull request by henryriveraCS**
# PR Summary
Fixing a typo where a period was used instead of space between "session" and "a".

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
close > reopend

**Review by msbemba — APPROVED** [review]
Approved. Regarding the errors, I do not see learn-validation VS Code extension in this PR.

File: docset/winserver2022-ps/grouppolicy/Copy-GPO.md [hunk 1]
```diff
@@ -70,6 +70,15 @@ The GPOs exist in the domain of the user that is running the session.
 
 ### Example 2: Copy a GPO from a domain to another domain
 
+```
```
Comment (msbemba) [inline docset/winserver2022-ps/grouppolicy/Copy-GPO.md:73]:
```powershell

Comment (msbemba) [inline docset/winserver2022-ps/grouppolicy/Copy-GPO.md:73]:
```powershell

File: docset/winserver2022-ps/grouppolicy/Copy-GPO.md [hunk 2]
```diff
@@ -70,6 +70,15 @@ The GPOs exist in the domain of the user that is running the session.
...
+PS C:\> Copy-GPO -SourceName "TestGpo1" -SourceDomain "test.contoso.com" TargetName "TestGpo1" -TargetDomain "sales.contoso.com"
```
Comment (msbemba) [inline docset/winserver2022-ps/grouppolicy/Copy-GPO.md:74]: Copy-GPO -SourceName "TestGpo1" -SourceDomain "test.contoso.com" TargetName "TestGpo1" -TargetDomain "sales.contoso.com"

**Review by msbemba — CHANGES_REQUESTED** [review]

**Review by msbemba — COMMENTED** [review]
Could you please change line 72 73 and 74 to below?

```powershell
Copy-GPO -SourceName "TestGpo1" -SourceDomain "test.contoso.com" TargetName "TestGpo1" -TargetDomain "sales.contoso.com"
```

**Review by msbemba — CHANGES_REQUESTED** [review]
Please change line 72,73 and 74 to fix the errors

```powershell
Copy-GPO -SourceName "TestGpo1" -SourceDomain "test.contoso.com" TargetName "TestGpo1" -TargetDomain "sales.contoso.com"
```

**Pull request closed without merging**